modify-publication/src/test/java/no/unit/nva/publication/modify/ModifyPublicationHandlerTest.java

```diff
@@ -48,11 +52,10 @@
     public static final String SOME_API_KEY = "some api key";
     public static final String HEADERS = "headers";
     public static final String BODY = "body";
-    public static final String PUBLICATION_JSON = "src/test/resources/publication.json";
     public static final String IDENTIFIER = "identifier";
     public static final String PATH_PARAMETERS = "pathParameters";
 
-    private ObjectMapper objectMapper = ModifyPublicationHandler.createObjectMapper();
+    private ObjectMapper objectMapper = new ObjectMapperConfig().objectMapper;
```

Comment:
objectMapper is static. Not need for creating a new Object. Make the OBjectMapeperConfig constructor private. You don't need a constructor for this class at this state.

Reply:
Will that fix the coverage issues?